docs/design/ovs-pipeline.md

```diff
@@ -586,6 +583,23 @@ is required), as well as for local Pod-to-Pod traffic.
 table=70, priority=0 actions=goto_table:80
 ```
 
+### L3DecTTLTable (71)
+This is the table to decrement TTL for L3 traffic on the source Node. It implements the following functionality:
+   
+ * TTL is decremented by one on the packet which is sent from a local Pod and destined
+   for a remote Pod. But for the traffic that enters OVS pipeline from the local gateway
```

Comment:
The first sentence uses "the packet", and this one uses "the traffic". Better to use one word, maybe "the packets".

Reply:
Updated.